res/keyboard/legacy_kbd_to_xml.py

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/python3
```

Comment:
No, this is a developer helper script, so I probably doesn't need to stay in this directory. I moved it to scripts.